contracts/access/Ownable.sol

```diff
@@ -16,11 +17,9 @@ import "../utils/Context.sol";
  * `onlyOwner`, which can be applied to your functions to restrict their use to
  * the owner.
  */
-abstract contract Ownable is Context {
+abstract contract Ownable is Context, IERC173 {
```

Comment:
Also the whole point of this would be to implement `supportsInterface` here, but not sure how welcome is it to add it (similar to how it is done in `token/ERC721/ERC721.sol` for example).